Library/Homebrew/cmd/update

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh # brew-update.sh
```

Comment:
Maybe go for `update.sh` to make it more obvious the difference between the shell and Ruby versions.